python/triton/compiler.py

```diff
@@ -1020,7 +1020,21 @@ def add_external_libs(mod, libs):
 def ttgir_to_llir(mod, extern_libs, compute_capability):
     if extern_libs:
         add_external_libs(mod, extern_libs)
-    return _triton.translate_triton_gpu_to_llvmir(mod, compute_capability)
+    ret = _triton.translate_triton_gpu_to_llvmir(mod, compute_capability)
+    # Check for shared memory size overflow.
+    shared_mem = _triton.get_shared_memory_size(mod)
+    device = torch.cuda.current_device()
```

Comment:
This PR breaks off-line tests on this line.
Because installer pytorch `torch.cuda.current_device()` throws exception.

We can

Reply:
Ah, ok, will try to fix it.

Reply:
Not sure, how to compute shred memory in that case.

Maybe pass (arch, triple, features) to this phase and infer size from them?
I mean arguments that are used in `llir_to_amdgcn_and_hsaco` transformation.